index.html

```diff
@@ -3361,9 +3361,12 @@ <h5 id="Regular Movement">Regular Movement</h5>
          <h5 id="Jumping Movement">Jumping Movement</h5>
          <p>
             Jumping movement works the same as regular movement except obstacles (such as
-            gaps, low obstacles, or small creatures) can be avoided on the path. What can or
-            cannot be jumped over is up to the GM’s discretion. The jump may require a roll
-            depending on the distance or height of the jump - this will use strength and/or
+            gaps, low obstacles, or small creatures) can be avoided on the path.
+            The distance that can be jumped in ft is equal to the creature's STR value 
```

Comment:
Probably a note that it can be rounded down for tile based combat instead of always rounding down; doesn't really make sense in world that a guy with 10 strength jumps exactly the same as a guy with 14 strength

Reply:
Ye absolutely good point